PWGJE/Tasks/hffragmentationfunction.cxx

```diff
@@ -234,7 +240,10 @@ struct HfFragmentationFunctionTask {
         axisDistance = RecoDecay::sqrtSumOfSquares(mcpjet.eta() - mcpd0cand.eta(), deltaPhi(mcpjet.phi(), mcpd0cand.phi()));
 
         // store data in MC detector level table
-        mcpdistJetTable(axisDistance, mcpjet.pt(), mcpjet.eta(), mcpjet.phi(), mcpd0cand.pt(), mcpd0cand.eta(), mcpd0cand.phi(), mcpd0cand.y(), mcpjet.has_matchedJetCand());
+        mcpdistJetTable(axisDistance,
+                        mcpjet.pt(), mcpjet.eta(), mcpjet.phi(),                                                                                                    // particle level jet
+                        mcpd0cand.pt(), mcpd0cand.eta(), mcpd0cand.phi(), mcpd0cand.y(), (mcpd0cand.originMcGen() == RecoDecay::OriginType::Prompt) ? true : false, // particle level D0
```

Comment:
```suggestion
                        mcpd0cand.pt(), mcpd0cand.eta(), mcpd0cand.phi(), mcpd0cand.y(), (mcpd0cand.originMcGen() == RecoDecay::OriginType::Prompt), // particle level D0
```

Reply:
Thanks for the suggestions and for approving it Vit.
Also sorry for the force-push, will take better care next time